metals/src/main/scala/scala/meta/internal/metals/NewFilesProvider.scala

```diff
@@ -182,11 +195,13 @@ class NewFilesProvider(
     )
   }
 
-  private def classTemplate(kind: String, name: String): String =
+  private def classTemplate(kind: String, name: String): String = {
+    val indent = "  "
     s"""|$kind $name {
-        |  
+        |$indent
```

Comment:
My editor is set to strip out whitespaces on save, so I've been struggling with this.

I think it's much clearer this way, anyway.

Reply:
I agree this is clearer. I also have my editor remove trailing whitespace on save, intellij has this enabled by default, so we should not rely on it in our tests

Reply:
I also like this as I get a warning in vim about trailing whitespaces all the time ha.